components/ConnectPage/ConnectPage.tsx

```diff
@@ -62,13 +62,15 @@ function ConnectPage() {
 
         {!address && (
           <WalletConnectButton>
-            <Image
-              src="/connect_wallet.png"
-              alt="WalletConnect"
-              width={400}
-              height={100}
-              loader={customLoader}
-            />
+            {() => (
```

Comment:
why are we updating all usages of `<WalletConnectButton />`?

Is there a way to make your intended change without breaking existing functionality  on other pages?